src/libxtp/atom.cc

```diff
@@ -17,10 +17,18 @@
  *
  */
 #include "votca/xtp/atom.h"
+#include <votca/tools/types.h>
 
 namespace votca {
 namespace xtp {
 
+  Atom::Atom(tools::StructureParameters params) : 
```

Comment:
I do not get this.

Reply:
Can you be a little more specific? By using a standardized constructor for different objects, one can more effectively use templates. You don't have to specialize the construction of a template method for each new class you support.